thirdparty/jansson/value.c

```diff
@@ -756,6 +760,68 @@ static json_t *json_string_copy(json_t *string)
     return json_string_nocheck(json_string_value(string));
 }
 
+// Given a json array, concat the elements using a delimiter
+w_string_t *w_string_implode(json_t *arr, const char *delim)
```

Comment:
Are you sure you need to do this? Seems like forward declarations + maybe hoisting `json_t` to a new file would work just as well.

I'm concerned because this pokes at internal details like the hash value. It would be nice if we contained all the code that dealt with that in one file.

(BTW if you're doing something non-obvious like this please expand on it in the commit message or in a comment.)